pages/common/mu.md

```diff
@@ -0,0 +1,32 @@
+# mu
+
+> Index and search emails from a local Maildir.
+> More information: <https://man.cx/mu(1)>.
+
+- Initialize the email database, optionally specifying the Maildir directory and email addresses belonging to self:
```

Comment:
We try to avoid `your`, `you`, etc.
```suggestion
- Initialize the email database, optionally specifying the Maildir directory and email addresses:
```